resources/shellHelperFunctions.sh

```diff
@@ -126,7 +126,7 @@ write_download_info() {
 
   FILE_EXTENSION="${BASE_FILE##*.}"
 
-  FILE_SIZE=$(stat -f"%z" "${BASE_PATH}/${BASE_FILE}")
+  FILE_SIZE=$(/usr/bin/stat -f"%z" "${BASE_PATH}/${BASE_FILE}")
```

Comment:
Does this also apply to?
https://github.com/keymanapp/keyman/blob/471ebcea1e7ca4559338122983371612ac3f167a/mac/Keyman4MacIM/write-download_info.sh#L84

Reply:
Fixed